docs/source/CONTRIBUTING.rst

```diff
@@ -105,15 +105,23 @@ Becoming a maintainer
 ~~~~~~~~~~~~~~~~~~~~~
 
 The project's maintainers will, from time-to-time, consider
-adding or removing a maintainer. An existing maintainer can submit a
-change set to the `maintainers <https://github.com/hyperledger/fabric/blob/master/MAINTAINERS.md>`__ file. A nominated
-Contributor may become a Maintainer by a majority approval of the proposal
+adding a maintainer, based on the following criteria:
+
+- Demonstrated track record of PR reviews (both quality and quantity of reviews)
+- Demonstrated thought leadership in the project
+- Demonstrated shepherding of project work and contributors
+
+An existing maintainer can submit a pull request to the
+`maintainers <https://github.com/hyperledger/fabric/blob/master/MAINTAINERS.md>`__ file.
+A nominated Contributor may become a Maintainer by a majority approval of the proposal
 by the existing Maintainers. Once approved, the change set is then merged
-and the individual is added to (or alternatively, removed from) the maintainers
-group. Maintainers may be removed by explicit resignation, for prolonged
+and the individual is added to the maintainers group.
+
+Maintainers may be removed by explicit resignation, for prolonged
 inactivity (3 or more months), or for some infraction of the `code of conduct
```

Comment:
Do we want to better define inactivity or just leave that up to the judgement of the maintainers at the time a proposal for removal?

Reply:
I've clarified as "e.g. 3 or more months with no review comments". This way there is at least one measurable criteria based on maintainer's primary responsibility, while still leaving the door open for judgement.